src/app/search/search.component.ts

```diff
@@ -378,20 +399,15 @@ export class SearchComponent implements OnInit {
   appendFilter(body: any, aggKey: string): any {
     this.filters.forEach((value: Set<string>, key: string) => {
       value.forEach(insideFilter => {
-        let modifiedInnerFilterValue = key;
-        // private_access is the only category we do not want modify
-        if (key !== 'private_access') {
-          modifiedInnerFilterValue = key.substring(0, 1) + key.substring(1).replace(/_/g, '.');
-        }
-        if (aggKey === key) {
-          // Return some garbage output because we've decided to append a filter, there's no turning back
+        if (aggKey === key && this.searchService.exclusiveFilters.indexOf(key) === -1) {
+          // Return some garbage filter because we've decided to append a filter, there's no turning back
           // return body;  // <--- this does not work
```

Comment:
I'll leave this in because I think most people who're trying to replace the garbage filter would try to use this line first.